### DISCUSSION TITLE
Filter project syntax to do: repo1 OR (repo2 AND label)

### DISCUSSION THREAD

**Discussion by sdruskat** [community/community#36761] [Projects and Issues] [answered]
In one of my projects, we pull issues from different (org-internal and org-external) repos into one project.

For one table view, I'd like to list issues from two repos, and for one of those, only issues with a specific label.

E.g., `repo:"external/repo1" OR (repo:"my/repo2" label:mylabel)`

Is this possible? And if not, could this be made possible?

Thanks!

**Comment by alexanderniebuhr** [comment, marked as answer]
According to the [docs](https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/filtering-projects), it is not possible yet.

> Using multiple filters will act as a logical AND filter. For example, label:bug status:"In progress" will return items with the bug label with the "In progress" status. Projects does not currently support logical OR filters across multiple fields.
